specification/security/resource-manager/Microsoft.Security/stable/2020-01-01/alerts.json

```diff
@@ -0,0 +1,999 @@
+      "additionalProperties": true,
+      "description": "Security alert",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "describes security alert properties.",
+          "$ref": "#/definitions/AlertProperties"
+        }
+      },
+      "allOf": [{
+        "$ref": "../../../common/v1/types.json#/definitions/Resource"
+      }]
+    },
+    "AlertEntity": {
+      "type": "object",
+      "additionalProperties": true,
+      "description": "Changing set of properties depending on the entity type.",
+      "properties": {
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Type of entity"
+        }
+      }
+    },
+    "AlertExtendedLinks": {
+      "type": "object",
+      "additionalProperties": true,
+      "description": "Links related to the alert"
+    },
+    "AlertExtendedProperties": {
+      "type": "object",
+      "additionalProperties": true,
+      "description": "Custom properties for the alert."
+    },
+    "AlertList": {
+      "type": "object",
+      "description": "List of security alerts",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "describes security alert properties.",
+          "items": {
+            "$ref": "#/definitions/Alert"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page."
+        }
+      }
+    },
+    "AlertProperties": {
+      "type": "object",
+      "description": "describes security alert properties.",
+      "properties": {
+        "alertType": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Unique identifier for the detection logic (all alert instances from the same detection logic will have the same alertType)."
+        },
+        "systemAlertId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Unique identifier for the alert."
+        },
+        "productComponentName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of Azure Security Center pricing tier which powering this alert. Learn more: https://docs.microsoft.com/en-us/azure/security-center/security-center-pricing"
+        },
+        "alertDisplayName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The display name of the alert."
+        },
+        "description": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Description of the suspicious activity that was detected."
+        },
+        "severity": {
+          "readOnly": true,
+          "type": "string",
+          "enum": [
+            "Informational",
+            "Low",
+            "Medium",
+            "High"
+          ],
+          "x-ms-enum": {
+            "name": "severity",
```

Comment:
The `severity` has already been defined [here](https://github.com/Azure/azure-rest-api-specs/blob/f01882efa05a57fb3b461ac5df3feaa1faca6e02/specification/security/resource-manager/Microsoft.Security/stable/2020-01-01/assessmentMetadata.json#L361), and it is inconsistency with the definition here (with different values). Maybe changing another word for this enum will work, or updating the existing swagger as well to make these two definitions align with each other.

Reply:
Your second here is not a link, can you fix this?

Reply:
The second `here` just means the code of this comment. Or if you insist, [here](https://github.com/nitsi/azure-rest-api-specs/blob/f01882efa05a57fb3b461ac5df3feaa1faca6e02/specification/security/resource-manager/Microsoft.Security/stable/2020-01-01/alerts.json#L653)

Reply:
Ok, got it.
I can't change it to match the assessment API:

1.  I want to keep as much backwards compatibility as passable.

Reply:
Ok, got it.
I can't change it to match the assessment API:
1) I want to keep as much backwards compatibility as passable with alerts 2019-01-01 [here](https://github.com/Azure/azure-rest-api-specs/blob/f01882efa05a57fb3b461ac5df3feaa1faca6e02/specification/security/resource-manager/Microsoft.Security/stable/2019-01-01/alerts.json#L575). The new version is to align our schema with our partners, today we export our alerts in this schema to ASI (Sentinel), log analytics, workspace automation and some other export features. So I wish to agile the severity to them. 
2) We have alerts with "information" severity while assessment don't

Reply:
I see now that the python code generation can't have 2 different definitions with the same name. I'll check with the assessment team to see if we can find a solution

Reply:
Fixed the issue